src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -208,8 +208,10 @@ public override IntermediateNode Visit(TexlBinaryOpNode node, IRTranslatorContex
                 switch (kind)
                 {
                     // Call Node Replacements:
-                    case BinaryOpKind.Power:
-                        binaryOpResult = new CallNode(context.GetIRContext(node), BuiltinFunctionsCore.Power, left, right);
+                    case BinaryOpKind.Power:
+                        var args = new List<IntermediateNode> { left, right };
+                        args = AttachArgPreprocessor(args, BuiltinFunctionsCore.Power, node, 2, context);
```

Comment:
There should probably be some code comments here explaining why the IR is doing explicit arg preprocessing here and not in other call or binary op nodes generated

Reply:
Added. Thanks!